test/test-schema.js

```diff
@@ -10,7 +10,8 @@ describe('Domain schema', () => {
         for (const region of regions) {
             for (const file of fs.readdirSync(`domains/${region}`)) {
                 const data = JSON.parse(fs.readFileSync(`domains/${region}/${file}`, 'utf8'))
-                assert.equal(ajv.validate(schema.domain, data), true, `${region}/${file}`)
+                ajv.validate(schema.domain, data), true, `${region}/${file}`
```

Comment:
Do we need ` true, `${region}/${file}`?

Reply:
Oops, looks like I forgot to remove that when refactoring the assertions. I'll make a PR to fix it.